projects/simple/src/Example/Project.hs

```diff
@@ -1,19 +1,49 @@
-module Example.Project (topEntity, plus) where
+-- @createDomain@ below generates a warning about orphan instances, but we like
+-- our code to be warning-free.
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Example.Project where
 
 import Clash.Prelude
 
--- | Add two numbers. Example:
+-- Create a domain with the frequency of your input clock. For this example we used
+-- 50 MHz.
+createDomain vSystem{vName="MyDomain", vPeriod=hzToPeriod 50e6}
```

Comment:
Heh I wrote this myself but regret it :-P

I couldn't think of a proper name and chose `MyDomain` but actually, I dislike these `My` things. Let's call it `Dom50`.